dotnet/src/Connectors/Connectors.Memory.Postgres/Connectors.Memory.Postgres.csproj

```diff
@@ -4,7 +4,7 @@
     <!-- THIS PROPERTY GROUP MUST COME FIRST -->
     <AssemblyName>Microsoft.SemanticKernel.Connectors.Postgres</AssemblyName>
     <RootNamespace>$(AssemblyName)</RootNamespace>
-    <TargetFrameworks>net8.0;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net8.0;netstandard2.0;net462</TargetFrameworks>
```

Comment:
`pgvector-dotnet` currently only supports `net6.0`. Add net462 directly, does it work?
`<TargetFrameworks>net6.0</TargetFrameworks>`

Reply:
The plugin currently targets PgVector [0.2.0](https://www.nuget.org/packages/Pgvector/0.2.0#dependencies-body-tab), which does target netstandard2.0. But we should indeed be targeting 0.3.0, which doesn't - I'll take a look.

Reply:
FYI submitted https://github.com/pgvector/pgvector-dotnet/pull/48 to pgvector.